### PR TITLE
Allow adding/overriding systemd service options

### PR DESCRIPTION
**Pull request by teddyphreak**
Add option service_systemd_options to allow adding / overriding systemd template options; see https://unix.stackexchange.com/questions/345595/how-to-set-ulimits-on-service-with-systemd/345596#345596 for an example use case
Add missing documentation for service_name parameter introduced with https://github.com/mhutter/ansible-docker-systemd-service/pull/18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,9 @@ This will create:
 * `enabled` (default: _yes_) - whether the service should be enabled
 * `masked` (default: _no_) - whether the service should be masked
 * `state` (default: _started_) - state the service should be in - set to
-  `absent` to remove the service.
+* `absent` to remove the service.
```
Comment (mhutter) [inline README.md]: Editing mistake; should go to the line above

**Review by mhutter — CHANGES_REQUESTED** [review]

**Comment by mhutter** [comment]
The template logic can probably be cleaned up a lot by collecting everything in one hash (merging with `service_systemd_options`) and simply iterating over it. But I'll look into this later

**Comment by teddyphreak** [comment]
README format should be addressed with e8295caab30af44d92538a1a00fb36b22ffcbfc3

File: README.md (hunk 1 above)
Comment (teddyphreak) [reply README.md]: README format should be addressed with e8295ca

**Review by mhutter — APPROVED** [review]

**Pull request merged**